src/scripts/experience/experience.js

```diff
@@ -33,6 +33,16 @@ define([
             this._showStats();
         }
 
+        var colophon = $('.colophon');
```

Comment:
NIT: I believe this is our typical style, but I could be wrong. I like to prefix jQuery object variables with a `$`. So this would be `$colophon`. Just makes things slightly more clear that you're dealing with a complex object and not a literal.